spec/features/step_by_step_navigation_spec.rb

```diff
@@ -9,6 +9,7 @@
     before do
       given_I_am_a_GDS_editor
       setup_publishing_api
+      stub_request(:put, %r{/publish-intent/}) # stub any PUT calls made by StepByStepScheduledPublishWorker
```

Comment:
There is a helper method that already does this: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api.rb#L37

Reply:
Thanks - done!